e2e/scenes/dashboards-suite/general-dashboards.spec.ts

```diff
@@ -30,6 +27,6 @@ describe('Dashboards', () => {
     // And the last panel should still be visible!
     // TODO: investigate scroll to on navigating back
     // e2e.components.Panels.Panel.title('Panel #50').should('be.visible');
-    e2e.components.Panels.Panel.title('Panel #1').should('be.visible');
+    // e2e.components.Panels.Panel.title('Panel #1').should('be.visible');
```

Comment:
haven't tested, but i think you can uncomment this 🤔 it's not really testing what it's supposed to test, but it should still pass